flocker_bb/builders/flocker_vagrant.py

```diff
@@ -218,6 +218,40 @@ def buildTutorialBox():
     return factory
 
 
+def run_client_installation_tests(configuration):
+    """Run the installation tests for clients."""
+    factory = getFlockerFactory()
+
+    if configuration.provider == 'vagrant':
```

Comment:
@tomprince Is this step necessary in the client tests? I added it because I was trying to match the cluster acceptance tests. Can you suggest any additions to the comment? Thanks.